core/nodejsActionBase/src/whisk.js

```diff
@@ -28,14 +28,7 @@ var request = require('request');
  */
 function Whisk(apihost, logger) {
 
-    // export __OW_API_HOST to environment
-    {
-        apihost = apihost || '';
-        var edgeHostParts = apihost.split(':');
-        var protocol = (edgeHostParts.length >= 2  &&  edgeHostParts[1] == '443') ? 'https' : 'http';
-        apihost = protocol + '://' + apihost;
-    }
-
+    apihost = 'https://' + apihost;
```

Comment:
This new code is not equivalent to the cold being replace when the input parameter `apihost` is falsy like `undefined`, `null`. 
In the old code it will default to type string `http://`
and in the new code it will default to `https://undefined` type string

If some other code tries to parse this string they will see the `undefined` in the uri.